core/src/read.jl

```diff
@@ -1395,11 +1397,52 @@ function Parameters(db::DB, config::Config)::Parameters
     return p
 end
 
-function get_ids(db::DB, nodetype)::Vector{Int32}
-    sql = "SELECT node_id FROM Node WHERE node_type = $(esc_id(nodetype)) ORDER BY node_id"
+function get_node_ints(db::DB, node_type)::Vector{Int32}
```

Comment:
Nit: This signature is very similar to get_node_ids and I almost thought you made a typo. `get_node_ids_int32` or similar?

Reply:
Ha yeah good point, done.